conans/tools.py

```diff
@@ -610,7 +611,7 @@ def __init__(self, runner=None, os_info=None, tool=None):
         self._is_up_to_date = False
         self._tool = tool or self._create_tool(os_info)
         self._tool._sudo_str = "sudo " if self._sudo else ""
-        self._tool._runner = runner or ConanRunner()
+        self._tool._runner = runner or ConanRunner(print_commands_to_output=True)
```

Comment:
I meant, if you need to use it, it is fine, but just change the default to True in the ``PkgUtilTool`` object, not for all other tools.